config/hsc/processCcd.py

```diff
@@ -19,3 +20,6 @@
 
 # Do not use NO_DATA pixels for fringe subtraction.
 config.isr.fringe.stats.badMaskPlanes=['SAT', 'NO_DATA']
+
+config.charImage.detectAndMeasure.measurement.algorithms["base_Jacobian"].pixelScale = 0.168
+config.calibrate.detectAndMeasure.measurement.algorithms["base_Jacobian"].pixelScale = 0.168
```

Comment:
Is it just "algorithms"->"plugins" you're referring too?  Otherwise I don't see anything obviously wrong here ("algorithms" is present for backwards compatibility, but we do prefer "plugins" now).